sql-migration-assistant/src/sql_migration_assistant/infra/app_serving_cluster_infra.py

```diff
@@ -1,9 +1,9 @@
-from databricks.sdk import WorkspaceClient
-from databricks.sdk.service.compute import ListClustersFilterBy, State, DataSecurityMode
-from databricks.labs.blueprint.tui import Prompts
-
 import logging
 
+from databricks.labs.blueprint.tui import Prompts
```

Comment:
what formatter did you use that rearranged these? DBlabs requires black formatting

Reply:
Ok, I'll use black then